apps/settings/js/panels/languages/languages.js

```diff
@@ -37,8 +40,24 @@ define(function(require) {
           f.localeFormat(d, _('shortTimeFormat'));
       }
     },
-    onInit: function(panel) {
+    showMoreLanguages: function() {
+      SettingsCache.getSettings(function(result) {
+        var version = result['deviceinfo.os'];
+        return new MozActivity({
+          name: 'marketplace-langpacks',
+          data: {
+            // Marketplace expects major.minor
+            version: version.split('.').slice(0, 2).join('.')
+          }
+        });
+      });
+    },
+    onInit: function(panel, elements) {
       this.panel = panel;
+      this.elements = elements;
+
+      this.elements.moreLanguages.onclick = this.showMoreLanguages;
+
       this.langSel =
```

Comment:
thanks for introducing `elements` here, can you help to update `this.langSel` to `this.elements.langSel` ? Thanks :)